### ISSUE TITLE
(Re-)Import of Generate Lightmap UVs option not updated

### ISSUE THREAD

**Issue by andgeno** [xavier150/Blender-For-UnrealEngine-Addons#53]
Hi,

It looks like the option `Generate Lightmap UVs` is never updated when I re-import my meshes either through UE4's content browser functionality or by running the generated Python script.

Could you please verify if this works on your end?

Thanks

**Comment by andgeno** [comment]
Some more information ...

I tried to manipulate the Asset's setting using the Python (REPL) console inside UE4 like this:

```
asset = unreal.find_asset('/Game/SM_TestMesh.SM_TestMesh')

asset.get_editor_property('asset_import_data').get_editor_property('generate_lightmap_u_vs')
// -> True

asset.get_editor_property('asset_import_data').set_editor_property('generate_lightmap_u_vs', False)
asset.get_editor_property('asset_import_data').get_editor_property('generate_lightmap_u_vs')`
// -> False
```


As you can see, the value changed from `True` to `False`. However, this change is **not** reflected in my static mesh asset when I take a look at it using the UE4's static mesh editor.

I suspect it might have to do with the `Apply changes` button that can be found in the **Details** panel in that editor window. Unfortunately, I haven't found a clue as to what would need to be called to run whatever code that this button triggers. 😞

**Comment by xavier150** [comment]
Thanks for the report! I try too on my side, We have two options Generate Lightmap UVs one for the import and one for the lods. (On my addon I set the options for the import but not for the lod0. So that work only for the first import) 
Like you I don't see code for this button but i continue to looking about.

**Comment by andgeno** [comment]
As a workaround, I wrote a little Blutility. But yeah, it woudl be nice if changes made in Blender in your addon would take effect on a re-import into Unreal.

**Comment by xavier150** [comment]
Yeah sure, I still never used Blutility, you can import and edit assets in content like the meshs ?

**Comment by andgeno** [comment]
They can be really powerful and are quite easy to create.
I will show you the Blutility that I made for this specific use-case:

![image](https://user-images.githubusercontent.com/2432608/78723012-ebceb000-792a-11ea-818e-faa8f08e3856.png)

**Comment by xavier150** [comment]
Thanks! Ooooh yeah, I will have to learn this

**Comment by xavier150** [comment]
Hello @andgeno 
Sorry I contact you after a long time, I fixed this issue for the next version and Push the Branch Dev.

The code to use is 


```
asset = unreal.find_asset('/Game/SM_TestMesh.SM_TestMesh')

asset.get_editor_property('asset_import_data').set_editor_property('generate_lightmap_u_vs', True)  # Import data
unreal.EditorStaticMeshLibrary.set_generate_lightmap_uv(asset, True)  # Build settings at lod
```

I will soon update the addon to 0.2.8

**Comment by xavier150** [comment]
Hey @andgeno I published the new [update 0.2.8](https://github.com/xavier150/Blender-For-UnrealEngine-Addons/releases/tag/v0.2.8) 
So I close this issue :)
